### ISSUE TITLE
terraform registry not syncing

### ISSUE THREAD

**Issue by azen1k5** [terraform-google-modules/terraform-google-lb-http#456]
### TL;DR

The latest release 12.0 is not available via the terraform registry.

### Expected behavior

  latest release can be downloaded via terraform registry

### Observed behavior

downloading fails with the following error:

```
Initializing modules...
╷
│ Error: Unresolvable module version constraint
│ 
│   on loadbalancer.tf line 9:
│    9: module "hono-loadbalancer-https" {
│ 
│ There is no available version of module "registry.terraform.io/terraform-google-modules/lb-http/google//modules/dynamic_backends" (loadbalancer.tf:9) which matches the given version constraint. The newest available version is 11.0.0.
````

### Terraform Configuration

```hcl
`
module "hono-loadbalancer-https" {                                       
  source  = "terraform-google-modules/lb-http/google//modules/dynamic_backends"                                                                                                      
  version = "~> 12.0"
                           
  // ....
}
````
```


### Terraform Version

```sh
`
Terraform v1.9.6
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v5.39.1
+ provider registry.terraform.io/hashicorp/google-beta v5.39.1
+ provider registry.terraform.io/hashicorp/random v3.6.3

Your version of Terraform is out of date! The latest version
is 1.9.7. You can update by downloading from https://www.terraform.io/downloads.html
````
```


### Additional information

_No response_

**Comment by bt-zdunning** [comment]
I think I'm running into a similar issue. I can't use `11.1.0` or `12.0.0`, but one of my colleagues can hit these versions just fine.

`11.0.0` works for me.

Error when attempting to pull `11.1.0`
```
INFO[0011] Downloading Terraform configurations from tfr:///terraform-google-modules/lb-http/google?version=11.1.0 into /.../.terragrunt-cache/CjiWXNdYEgNBuvU5ymWa7EtcKds/M_PfLy0pF3XeXUItqVYgAXWn6A4
ERRO[0011] downloading source url tfr:///terraform-google-modules/lb-http/google?version=11.1.0
1 error occurred:
        * error downloading 'tfr:///terraform-google-modules/lb-http/google?version=11.1.0': Error downloading module from https://registry.terraform.io/v1/modules/terraform-google-modules/lb-http/google/11.1.0/download: error receiving HTTP data

ERRO[0011] Unable to determine underlying exit code, so Terragrunt will exit with error code 1
```

Error when attempting to pull `12.0.0`
```
INFO[0009] Downloading Terraform configurations from tfr:///terraform-google-modules/lb-http/google?version=12.0.0 into /.../.terragrunt-cache/CjiWXNdYEgNBuvU5ymWa7EtcKds/M_PfLy0pF3XeXUItqVYgAXWn6A4
ERRO[0009] downloading source url tfr:///terraform-google-modules/lb-http/google?version=12.0.0
1 error occurred:
        * error downloading 'tfr:///terraform-google-modules/lb-http/google?version=12.0.0': Error downloading module from https://registry.terraform.io/v1/modules/terraform-google-modules/lb-http/google/12.0.0/download: error receiving HTTP data

ERRO[0009] Unable to determine underlying exit code, so Terragrunt will exit with error code 1
```

**Comment by imrannayer** [comment]
Fixed

**Comment by imrannayer** [comment]
@g-awmalik @bharathkkb this is another repo not syncing with registry automatically.
